platform/ui/src/components/Table/Table.mdx

```diff
@@ -0,0 +1,111 @@
+---
+name: Table
+menu: Components
+route: components/table
+---
+
+import { Playground, Props, Link } from 'docz';
+import { Table, TableHead, TableBody, TableRow, TableCell } from '@ohif/ui';
+
+# Table
+
+Tables display sets of data.
+
+## Import
+
+```javascript
+import { Table } from '@ohif/ui';
```

Comment:
Is it weird to use an already existing HTML tag here? Should we do something that clearly sets them apart?

Reply:
`<Table>` (React Component) looks a like like `<table>`, the native HTML DOM element.
Should we rename it to something with a vendor prefix, or a two-word name?

`<OhifTable>` or `<SimpleTable>`

Reply:
I think it's ok to have a `Table`, it's capitalized, implying that it's a React component instead of a pure HTML element. This is also how many other component libraries do. What do you think we can do here?

@galelis what's your opinion?

Reply:
If this is common in other libraries, that's cool with me.